### PR TITLE
translating the Portuguese basic information page

### PR DESCRIPTION
**Pull request by ElisaCapololo**
…with Bolt For Javascript

This PR is a translation work.

## Changes

- Translation from English to Portuguese of the basic information page. Issue #85

## Screenshots

(prefer animated gif)
Screenshot

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

Fixes #

### CODE REVIEW

**Comment by antonio-pedro99** [comment]
@ElisaCapololo please mention the issue number in the descriptions, like fixes #93

**Comment by Busayo-ojo** [comment]
Hi @ElisaCapololo, thanks for working on this! Kindly sign this off so I can merge this pull request.

You can see this article for clarity on how to sign off your comits https://www.secondstate.io/articles/dco/

**Comment by ElisaCapololo** [comment]
Hello Busayo,

I believe I have already managed to add the DCO.

Thanks,
Elisa

File: Translations/Portuguese/Basic Information.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
 This is the link  to the slack bot wiki basic information page to be translated.
-[Basic Information](https://github.com/chaoss/chaoss-slack-bot/wiki/Basic-Information)
\ No newline at end of file
+[Basic Information](https://github.com/chaoss/chaoss-slack-bot/wiki/Basic-Information)
+
+> ### Informação básica
+
+> #### Funções do bot
```
Comment (antonio-pedro99) [inline Translations/Portuguese/Basic Information.md:6]:
```suggestion
> #### Funcionalidades do bot
```

File: Translations/Portuguese/Basic Information.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
 This is the link  to the slack bot wiki basic information page to be translated.
-[Basic Information](https://github.com/chaoss/chaoss-slack-bot/wiki/Basic-Information)
\ No newline at end of file
+[Basic Information](https://github.com/chaoss/chaoss-slack-bot/wiki/Basic-Information)
+
+> ### Informação básica
```
Comment (antonio-pedro99) [inline Translations/Portuguese/Basic Information.md:4]:
```suggestion
> ### Informações básica
```

**Review by antonio-pedro99 — APPROVED** [review]
Thanks for you hardwork on this @ElisaCapololo 
I left some comments, please address them and we are fine to merge this :)

**Review by antonio-pedro99 — APPROVED** [review]

File: Translations/Portuguese/Basic Information.md (hunk 2 above)
Comment (ElisaCapololo) [reply Translations/Portuguese/Basic Information.md]: I have already made the recommended changes

**Review by aosccode — CHANGES_REQUESTED** [review]
Hi @ElisaCapololo, there are some merge conflicts.
Can you fix them? after this I think we are ready to go

**Pull request merged**